lib/blacklight/engine.rb

```diff
@@ -45,7 +45,7 @@ class Engine < Rails::Engine
 
     config.routes = OpenStruct.new
     # Set identifier_constraint to enforce a format for the document identifiers
-    config.routes.identifier_constraint = /[\w-]+/
+    config.routes.identifier_constraint = /.+/
```

Comment:
Won't this break anyone using URLs with extensions?

Reply:
Explain what you mean by that?

Reply:
Keep in mind this is only for the "track" route: https://github.com/projectblacklight/blacklight/blob/v6.20.0/config/routes.rb#L10